vignettes/Accept-organism-for-GenomeInfoDb.Rmd

```diff
@@ -0,0 +1,60 @@
+---
+title: "Submitting your organism to GenomeInfoDb"
+author:
+- name: "Sonali Arora"
+- name: "H. Khan"
+  affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+package: GenomeInfoDb
+vignette: >
+  %\VignetteIndexEntry{Submitting your organism to GenomeInfoDb}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document
+---
+
+This document is meant for package developers wanting to submit an organism which does not already exist in [*GenomeInfoDb*](http://bioconductor.org/packages/release/bioc/html/GenomeInfoDb.html), to be a part of [*GenomeInfoDb*](http://bioconductor.org/packages/release/bioc/html/GenomeInfoDb.html).
+
+# Background
+
+*GenomeInfoDb* provides a multitude of functions to subset and access seqName mappings and styles associated with an organism. One can use the supplementary manual "GenomeInfoDb.pdf" to see such functions. Alternately if there is no support for an organism for one's choice, one can submit a tab delimited file (as detailed by the sections below) to request their favorite organism to become a part of GenomeInfoDb package.
+
+# Support for existing organisms
+
+GenomeInfoDb already provides support for a multitude of organisms, to see a detailed list of organism and the supported seqname styles associated with those organism, one can simply run :
+
+```{r message=FALSE}
+library(GenomeInfoDb)
+names(genomeStyles())
+```
+
+# File format for new organism
+
+If your favorite organism does not exist in the above list, one can submit a tab-delimited file in the following format and request for your organism to be added to [*GenomeInfoDb*](http://bioconductor.org/packages/release/bioc/html/GenomeInfoDb.html)
+
+1.  The file name should be in the following format:
```

Comment:
After "in the following format", please add the following text to match the original vignette:  ‘genus\_{}species.txt’

Reply:
I have updated the changes